manifests/overlays/minimal/README.md

```diff
@@ -0,0 +1,4 @@
+## Dashboard Minimal overlay
```

Comment:
I don't see the harm on having a minimal overlay, but maybe we would want to have this in other PR?

cc @andrewballantyne @christianvogt what do you think? Not sure if we should stick this, which is related to a deployment of our repo with this same PR.

Reply:
This is a neat idea... I'm not against new overlays like this. @cam-garrison was this something you used in your testing/coding?

Reply:
Yeah, this can be removed. It is anyway not possible to combine overlays like it was with `KfDef` magic, so better to make it separate and shrink this PR.

Reply:
Yes, we used this for testing/running in CRC specifically. +1 on bartosz's point though, can be removed especially since in v2 operator it is not as easy to apply multiple overlays.